monix-eval/shared/src/test/scala/monix/eval/TypeClassLawsForTaskWithCallbackSuite.scala

```diff
@@ -29,6 +30,8 @@ import scala.concurrent.Promise
   * use of Task's `runAsync(callback)`.
   */
 object TypeClassLawsForTaskWithCallbackSuite extends BaseLawsSuite {
+  implicit val ap: Applicative[Task.Par] = CatsParallelForTask.applicative
```

Comment:
Ah, wait — so here we need a special equality defined for `Task.Par`, just like we have for `Task`. The purpose of this test suite is to test task's run-loop by using the `runAsync(callback)` method, instead of the normal `runAsync: CancelableFuture` that we are using in these tests.